src/client/task.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"context"
+	"io"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/grpcutil"
+	"github.com/pachyderm/pachyderm/v2/src/taskapi"
+)
+
+// ListTask lists tasks in the given namespace and group
+func (c APIClient) ListTask(namespace string) ([]*taskapi.TaskInfo, error) {
```

Comment:
I tend to prefer doing the gRPC scrub in a defer so I can just return err in the actual function.